Library/Homebrew/language/python.rb

```diff
@@ -66,6 +73,7 @@ def self.setup_install_args(prefix)
         --no-user-cfg
         install
         --prefix=#{prefix}
+        --install-scripts=#{prefix}/bin
```

Comment:
`#{bin}`?

Reply:
That variable doesn't exist in this context. `prefix` is passed from the caller (usually the formula for a more advanced build) and typically goes in the formula equivalent of `libexec` or similar.

Reply:
I assumed it would exist when this variable would actually be called? We use `buildpath` in this file, for example, which obviously isn't defined at this point. Could be wrong, it's 5am and I definitely should be asleep 😅.

Reply:
Yeah, it does exist while called, but this function doesn't have access to it. `prefix` is passed in by the formula, and it can be anything path-like. Check out e.g. [sphinx-doc.rb](https://github.com/Homebrew/homebrew-core/blob/d9908480053b8ed2db53d44e485bd669baa05970/Formula/sphinx-doc.rb#L142) for what I mean. `libexec` (in the formula's context) turns into `prefix` here.

`buildpath` works because it's part of the `Virtualenv` mixin class, which I believe would have access to its cohabitants variables like `bin` etc.